package.json

```diff
@@ -76,8 +76,8 @@
     "lodash": "^4.17.11",
     "long": "^4.0.0",
     "mime": "^2.4.0",
-    "minimed-connect-to-nightscout": "^1.1.1",
-    "moment": "^2.22.2",
+    "minimed-connect-to-nightscout": "https://github.com/szymjaw/minimed-connect-to-nightscout.git#dev-servers",
```

Comment:
we only allow released versions (due to npm security hashes).

Reply:
I incorporated that line from @szymjaw 's changes in #4190, assuming they knew what they were doing; perhaps they could comment/fix.

If we don't get a response before you need to merge, will just revert this to the original line.

Reply:
If the authors of that PR don't upgrade `minimed-connect-to-nightscout` in the coming days, please revert to 1.1.1.

Reply:
I have no contact with @mddub from several monts. I do not want to assign myself to authorship, I only modified the code. 
I created npm package "minimed-connect-to-nightscout-szymjaw". I created dev-guardian-connect branch for cgm-remote-monitor and it working on heroku.

Reply:
Where does this leave us, @PieterGit ?

Reply:
@unsoluble please revert minimed changes in this PR. then I can merge to dev and release
We will upgrade minimed-connect-to-nightscout in 0.11.x or 0.12.x when https://github.com/mddub/minimed-connect-to-nightscout/pull/11 is finished by @szymjaw

Reply:
@PieterGit Turns out the support for EU accounts wasn't ever present in the previous version of minimed-connect-to-nightscout; that was added in @szymjaw 's version. Have updated the docs to indicate this, but we should attempt to include this sooner than later.